apps/src/templates/lessonOverview/ResourceList.jsx

```diff
@@ -16,10 +22,47 @@ export default class ResourceList extends Component {
     }
   };
 
+  recordResourceDownload = resource => {
+    firehoseClient.putRecord(
+      {
+        study:
+          this.props.pageType === 'resources-rollup'
+            ? 'rollup-pages'
+            : 'lesson-plan',
+        study_group: this.props.pageType,
+        event: 'download-resource',
+        data_json: JSON.stringify({
+          resourceId: resource.id
+        })
+      },
+      {includeUserId: true}
+    );
+  };
+
+  recordResourceOpened = resource => {
+    firehoseClient.putRecord(
+      {
+        study:
+          this.props.pageType === 'resources-rollup'
+            ? 'rollup-pages'
+            : 'lesson-plan',
+        study_group: this.props.pageType,
+        event: 'open-resource',
+        data_json: JSON.stringify({
+          resourceId: resource.id
+        })
+      },
+      {includeUserId: true}
+    );
+  };
+
   createResourceListItem = resource => (
     <li key={resource.key}>
       <a
         href={this.normalizeUrl(resource.url)}
+        onClick={() => {
+          this.recordResourceOpened(resource);
+        }}
```

Comment:
like with https://github.com/code-dot-org/code-dot-org/pull/40071, it doesn't really work to mix onClick functionality with standard links, particularly when the onClick is making an HTTP request

Reply:
Alright will make the same update here :)